_layouts/release.html

```diff
@@ -5,7 +5,11 @@
 <article class="release release">
     <header class="release_header">
         <h1 class="release_title">{{ page.title }}</h1>
+        {% if page.release_tag == nil %}
         <p class="release_meta"><time datetime="{{ page.date | date_to_xmlschema }}" itemprop="datePublished">{{ page.date | date: "%A, %Y-%m-%d" }}</time></p>
+        {% else %}
+        <p class="release_meta">Release: {{ page.release_tag }}</p>
+        {% endif %}
```

Comment:
OH> I prefer to avoid keeping those ifs for "backwards compatibility".

+1. 

Also, do we really need to keep backwards compatibility? Do we have a use case for recreating the release notes of an old version? IMHO we are throwing away effort/time with this requirement.